src/sentry/search/snuba/backend.py

```diff
@@ -234,7 +237,7 @@ def assigned_or_suggested_filter(
         owned_by_me = Q(
             **{
                 f"{field_filter}__in": GroupOwner.objects.filter(
-                    Q(user__in=users) | Q(team_id__in=team_ids),
+                    Q(user_id__in=users) | Q(team_id__in=team_ids),
```

Comment:
Should this be `user_ids`?

Reply:
It actually works either way (plural vs singular), but I wanted to be consistent (other IN queries in this file also use singular casing)

Reply:
I actually meant the `users` variable. Shouldn't that be `user_ids`? I'm not sure if django is magically translating it for us here.

Reply:
You are correct on both accounts and I'll adjust.

Reply:
https://github.com/getsentry/sentry/pull/44391/commits/d4f0467c4b09ba3e69bc64b3f4cd9d19859dc95a